wfe/web-front-end_test.go

```diff
@@ -151,10 +250,16 @@ func signRequest(t *testing.T, req string) string {
 	return ret
 }
 
+func TestSetup(t *testing.T) {
+	// Very first test to run, now everyone can enjoy HandlePaths()
+	// for header checks.
+	wfe.HandlePaths()
```

Comment:
If it's possible, this would be better done as a function that returns a configured WFE, as is done with the [SA tests](https://github.com/letsencrypt/boulder/blob/master/sa/storage-authority_test.go#L26).  That way, the tests can be maximally independent of each other.